src/uct/base/uct_iface.h

```diff
@@ -33,6 +33,8 @@
  * is extended by a system namespace information */
 #define UCT_IFACE_LOCAL_ADDR_FLAG_NS UCS_BIT(63)
 
+/* UCT IFACE with AM SHORT must support a minimal number of iov */
+#define UCT_IFACE_AM_SHORT_MIN_IOV 4
```

Comment:
isn't it just for UD? why need 4 IOV on RC/DC/SHM/...?
this is related to internal UD implementation and not UCT in general

Reply:
The current user is UD, but this is the count that is passed to `uct_ep_am_short_iov()` so I thought it made sense to put it in base, but not in API, could be moved though.